dts/arm/nordic/nrf54l15_cpuapp.dtsi

```diff
@@ -4,88 +4,48 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <arm/armv8-m.dtsi>
-#include <nordic/nrf_common.dtsi>
-#include <nordic/nrf54l_common.dtsi>
+#include <nordic/nrf54l15.dtsi>
 
-/ {
-	cpus {
-		#address-cells = <1>;
-		#size-cells = <0>;
-
-		cpu: cpu@0 {
-			clock-frequency = <DT_FREQ_M(128)>;
-			device_type = "cpu";
-			compatible = "arm,cortex-m33f";
-			reg = <0>;
-			#address-cells = <1>;
-			#size-cells = <1>;
-			itm: itm@e0000000 {
-				compatible = "arm,armv8m-itm";
-				reg = <0xe0000000 0x1000>;
-				swo-ref-frequency = <DT_FREQ_M(128)>;
-			};
-		};
-	};
+cpu: &cpuapp {};
+systick: &cpuapp_systick {};
+nvic: &cpuapp_nvic {};
+cpuflpr_vevif: &cpuflpr_vevif_remote {};
 
-	clocks {
-		lfxo: lfxo {
-			compatible = "nordic,nrf-lfxo";
-			#clock-cells = <0>;
-			clock-frequency = <32768>;
-		};
-
-		hfxo: hfxo {
-			compatible = "nordic,nrf-hfxo";
-			#clock-cells = <0>;
-			clock-frequency = <DT_FREQ_M(32)>;
-		};
-	};
+/delete-node/ &cpuflpr;
+/delete-node/ &cpuflpr_rram;
+/delete-node/ &cpuflpr_sram;
+/delete-node/ &cpuflpr_clic;
 
+/ {
 	soc {
-		uicr: uicr@ffd000 {
-			compatible = "nordic,nrf-uicr";
-			reg = <0xffd000 0x1000>;
-		};
-
-		ficr: ficr@ffc000 {
-			compatible = "nordic,nrf-ficr";
-			reg = <0xffc000 0x1000>;
-			#nordic,ficr-cells = <1>;
-		};
+		compatible = "simple-bus";
+		interrupt-parent = <&cpuapp_nvic>;
+		ranges;
+	};
+};
 
-		sram0: memory@20000000 {
-			compatible = "mmio-sram";
-			reg = <0x20000000 DT_SIZE_K(256)>;
-		};
+&cpuapp_ppb {
+	compatible = "simple-bus";
+	ranges;
+};
 
-		peripheral@50000000 {
-			#address-cells = <1>;
-			#size-cells = <1>;
-			ranges = <0x0 0x50000000 0x10000000>;
+&grtc {
+	interrupts = <228 NRF_DEFAULT_IRQ_PRIORITY>,
+			<229 NRF_DEFAULT_IRQ_PRIORITY>; /* MPSL uses the GRTC_3_IRQn */
```

Comment:
This comment should probably be removed, there is no MPSL upstream.

Reply:
Or maybe rephrased to sth like "reserved for Zero Latency IRQs"